setup.py

```diff
@@ -2,7 +2,7 @@
 """Python Ring Door Bell setup script."""
 from setuptools import setup
 
-_VERSION = '0.2.9'
+_VERSION = '0.6.2'
```

Comment:
Should this be bumped to `0.7.0`?

```suggestion
_VERSION = '0.7.0'
```